documentation/API_documentation/QoSProfile_Mapping_Table.md

```diff
@@ -14,3 +15,115 @@ It is generally assumed that there exists an agreement between the API Invoker a
 
 **Note:**
 The QOS profile labels in the table can be used within Camara for validating the QoD APIs.
+
+## ExtendedQosProfile
```

Comment:
Definitions below don't match the current definitions in the API OAS definition.